logo/index.md

```diff
@@ -1,110 +1,71 @@
 ---
-title: Logo Contest
+title: The GAP Logo
 layout: default
 has_children: true
 permalink: /logo/
 ---
 
-# GAP logo contest
+# The GAP Logo
 
-This contest is about designing an official logo for the GAP computer algebra system, to be used on our website <https://www.gap-system.org> and also for research posters, application icons, stickers, mugs, and so on. The logo may use colors (and the color scheme of the website can be adapted to match the logo), but a B&W version should also be created.
+On this page you can learn a bit more about the GAP Logo: how it was selected,
+what it signifies, and also where to download it for use in e.g. presentations
+or on posters.
 
-We are looking for a logo that reflects what GAP is. To that end we provide some extra information and inspiration below, after the rules of the contest.
+We also plan to eventually offer some merchandise (stickers, T-shirts etc.)
```

Comment:
Maybe say something like:
Eventually, it will be possible to buy merchandise (stickers, t-shirts etc.) with the logo.

Reply:
```suggestion
Eventually, it will be possible to buy merchandise (stickers, t-shirts etc.) with the logo.
```